src/services/telemetryService.ts

```diff
@@ -98,26 +104,25 @@ export interface UserFlowTelemetry extends TelemetryOptions {
 }
 
 export interface ITelemetryService {
-	sendTelemetryEvent(eventName: string, properties?: TelemetryEventProperties): void;
-	sendRawTelemetryEvent(eventName: string, properties?: RawTelemetryEventProperties): void;
-	sendTelemetryException(error: Error, properties?: TelemetryEventProperties): void;
+	sendTelemetryEvent(gitpodHost: string, eventName: string, properties?: TelemetryEventProperties): void;
+	sendRawTelemetryEvent(gitpodHost: string, eventName: string, properties?: RawTelemetryEventProperties): void;
+	sendTelemetryException(gitpodHost: string, error: Error, properties?: TelemetryEventProperties): void;
```

Comment:
would it be possible to add gitpodHost automatically in the `${gitpodHost}/analytics` endpoint as the body is just json instead of doing it in every call? for development it doesn't matter as we know which host we are testing manually